src/main/groovy/edu/oregonstate/mist/api/InfoResource.groovy

```diff
@@ -14,6 +14,21 @@ import javax.ws.rs.core.Response
 class InfoResource extends Resource {
     private Info info = new Info()
 
+    public InfoResource() {
+        def stream = this.class.getResourceAsStream('/build.properties')
+        if (stream == null) {
+            throw new Exception("couldn't load build.properties")
+        }
```

Comment:
A couple of things: 
# the check can be changed to: if (!stream) to make the code shorter
# have you tested whether the exception will prevent the startup of the program? An error should be thrown, but probably not prevent the application from running